toolchain/gcc.py

```diff
@@ -89,7 +93,7 @@ def test(self):
             process_kwargs={'ignore_status': True})
         self.summary = ret.stdout.splitlines()
```

Comment:
Better to add `decode()` here so that the other part of code can still remain the same.